packages/page-pattern-modal/src/components/page-pattern-modal.tsx

```diff
@@ -153,7 +154,18 @@ class PagePatternModal extends Component< PagePatternModalProps, PagePatternModa
 		this.setState( { selectedCategory } );
 	};
 
-	closeModal = () => {
+	closeModal = ( event: FocusEvent ) => {
+		// As of Gutenberg 13.1, the editor will auto-focus on the title block
+		// automatically. See: https://github.com/WordPress/gutenberg/pull/40195.
+		// This ends up triggering an `onBlur` event on the Modal that causes it
+		// to close just after the editor loads. To circunvent this, we check if
```

Comment:
nit/typo: circuNvent instead of circuMvent

Reply:
Thanks for catching that!